File: 프로그래머스/unrated/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    score.sort()
    cnt=0
    while len(score) >= m:
        result=[]
        for i in range (m):
            result.append(score.pop())
        cnt+=(min(result)*m)
        if len(score)<m:
            break
    return cnt

File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    score = score.sort((a,b)=>b-a)
    var len=score.length
    var result=0
    var n=0
    while(len>=m){
        var num=score.slice(n,m+n)
        result+=Math.min(...num)*m
        len-=m
        n+=m
    }
    return result
}